packages/did-core-test-server/suites/did-resolution/did-resolution.js

```diff
@@ -186,115 +175,122 @@ const didResolutionTests = (execution, expectedOutcome) => {
+        })
+      });
+      it('Each equivalentId DID value MUST be produced by, and a form of, the same DID Method as the id property value.', async () => {
+        if (didDocument) {
+          didDocumentMetadata['equivalentId'].forEach((equivalentId) => {
+            expect(utils.parseDidMethod(equivalentId)).toBe(utils.parseDidMethod(didDocument['id']));
           });
-          // As discussed on the 2021-04-13 DID WG topic call, the following test can be skipped (see https://www.w3.org/2019/did-wg/Meetings/Minutes/2021-04-13-did-topic)
-          it.skip('A conforming DID Method specification MUST guarantee that each equivalentId value is logically equivalent to the id property value.');
-          // As discussed on the 2021-04-13 DID WG topic call, the following test can be skipped (see https://www.w3.org/2019/did-wg/Meetings/Minutes/2021-04-13-did-topic)
-          it.skip('equivalentId is a much stronger form of equivalence than alsoKnownAs because the equivalence MUST be guaranteed by the governing DID method.');
         }
       });
-      describe('canonicalId', () => {
-        // TODO: According to the spec, 'canonicalId' is currently not optional, even though that's definitely what the WG intended - see https://github.com/w3c/did-core/issues/717.
-        if (didDocumentMetadata.hasOwnProperty('canonicalId')) {
-          it('The value of canonicalId MUST be a string that conforms to the rules in Section § 3.1 DID Syntax.', async () => {
-            expectConformantDid(didDocumentMetadata['canonicalId']);
-          });
-          it('A canonicalId value MUST be produced by, and a form of, the same DID Method as the id property value.', async () => {
-            // TODO: This will currently only work if the 'resolve' function is called, not 'resolveRepresentation'.
-            expect(parseDidMethod(didDocumentMetadata['canonicalId'])).toBe(parseDidMethod(didDocument['id']));
-          });
-          // As discussed on the 2021-04-13 DID WG topic call, the following test can be skipped (see https://www.w3.org/2019/did-wg/Meetings/Minutes/2021-04-13-did-topic)
-          it.skip('A conforming DID Method specification MUST guarantee that the canonicalId value is logically equivalent to the id property value.');
+      // As discussed on the 2021-04-13 DID WG topic call, the following test can be skipped (see https://www.w3.org/2019/did-wg/Meetings/Minutes/2021-04-13-did-topic)
+      it.skip('A conforming DID Method specification MUST guarantee that each equivalentId value is logically equivalent to the id property value.', () => {
+      });
+      // As discussed on the 2021-04-13 DID WG topic call, the following test can be skipped (see https://www.w3.org/2019/did-wg/Meetings/Minutes/2021-04-13-did-topic)
+      it.skip('equivalentId is a much stronger form of equivalence than alsoKnownAs because the equivalence MUST be guaranteed by the governing DID method.', () => {
+      });
```

Comment:
```suggestion
```

I suggest we just not have .skip() and .todo()s in the test suite. We haven't done that elsewhere and it'll be more work than necessary at this point. We do have an accouting/audit of this in the issues that have been raised. I suggest that doing so should be enough for the purposes of CR/PR/REC.